src/docbuilder/rustwide_builder.rs

```diff
@@ -237,6 +237,14 @@ impl RustwideBuilder {
                 for (&file, versioned) in files {
                     let segments = file.rsplitn(2, '.').collect::<Vec<_>>();
                     let file_name = if versioned {
+                        if segments[0] == "js"
+                            && ["crates", "search-index"].iter().any(|x| *x == segments[1])
```

Comment:
Couldn't you just remove `crates.js` from the `ESSENTIAL_FILES_VERSIONED` list above?

Reply:
This will also cause https://github.com/rust-lang/docs.rs/issues/1300 to reoccur, we might need to generate a `crates.js` that sets an empty array or something (or really it should come from each doc build, not the essential files, to support multi-crate packages in the future).

Reply:
It should definitely come from each doc build.